### ISSUE TITLE
[Accessibility]A11y_VSCode_ViewExtensions_Keyboard:Keyboard focus is not visible on install button for solarized dark theme.

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/vscode#51993]
**Environment Details:**
VSCode Version : 1.24.1
OS Version : Win10
**Additional Details:**
Keyboard,MAS2.4.7
**Repro Steps:**
1. Launch VS Code.
2.Navigate to View(Alt+V) menu option and select "Extensions" from dropdown.
3. Check the keyboard focus on the screen.

**Actual:**
Keyboard focus is not visible on "install" button for solarized dark theme.

**Expected:**
Keyboard focus should visible on "install" button for solarized dark theme.
Same issue observed on some other theme also.
**Recommendations:**
Refer below link which is repository of bug fixes code snippets:
[https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx](url)

**User Impact:**
Keyboard User will not able to access the control which is accessible by mouse.
**MAS Reference:**
[https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={3b98b04f-4fab-4d3e-81af-a0906a34849c}](url)
Does this issue occur when all extensions are disabled?: Yes
**Attachment for reference:**
[[Accessibility]A11y_VSCode_ViewExtensions_Keyboard.pptx](https://github.com/Microsoft/vscode/files/2105480/Accessibility.A11y_VSCode_ViewExtensions_Keyboard.pptx)

**Comment by RMacfarlane** [comment]
Duplicate of https://github.com/Microsoft/vscode/issues/50883
